pkgs/development/interpreters/spidermonkey/bug-1526653.patch

```diff
@@ -0,0 +1,11 @@
+--- firefox-68.4.2/js/src/wasm/WasmSignalHandlers.cpp.orig	2020-04-30 06:35:16.243047468 +0900
++++ firefox-68.4.2/js/src/wasm/WasmSignalHandlers.cpp	2020-04-30 06:35:44.110598805 +0900
+@@ -243,7 +243,7 @@
+ // If you run into compile problems on a tier-3 platform, you can disable the
+ // emulation here.
+ 
+-#if defined(__linux__) && defined(__arm__)
++#if 0
```

Comment:
Could we not check for `__ANDROID__`? It would make it clearer what’s going on, and mean that if somebody (in theory) built this for Android they’d get the right thing.